src/applications/vaos/new-appointment/components/CommunityCareProviderSelectionPage/index.jsx

```diff
@@ -30,35 +31,59 @@ const initialSchema = {
 };
 
 const pageKey = 'ccPreferences';
-const pageTitle = 'Tell us your community care preferences';
+let pageTitle = 'Tell us your community care preferences';
 
 export default function CommunityCareProviderSelectionPage() {
   const dispatch = useDispatch();
   const { data, pageChangeInProgress, schema, showCCIterations } = useSelector(
     state => getFormPageInfo(state, pageKey),
     shallowEqual,
   );
+  const residentialAddress = useSelector(
+    state => selectVAPResidentialAddress(state),
+    shallowEqual,
+  );
   const history = useHistory();
 
+  if (showCCIterations && residentialAddress) {
+    // TODO: Should the type of care name be lower case???
+    pageTitle = `Request a ${getTypeOfCare(data).name} provider`;
```

Comment:
May want to pull in the latest, I think @jenniemc made this copy change. However, that change does need to be updated to use the lowerCase function in formatters.

Reply:
Done